staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/aggregator/aggregator.go

```diff
@@ -161,7 +164,7 @@ func (s *specProxier) UpdateAPIServiceSpec(apiServiceName string) error {
 func (s *specProxier) updateAPIServiceSpecLocked(apiServiceName string) error {
 	apiService, exists := s.apiServiceInfo[apiServiceName]
 	if !exists {
-		return fmt.Errorf("APIService %s does not exist for update", apiServiceName)
+		return ErrAPIServiceNotFound
```

Comment:
we exchange a nice error message with context with one without context. Looks suspicious.

Reply:
This was to be consistent with the handling in openapi v2: https://github.com/kubernetes/kubernetes/blob/1144c85a929bb7a7d30e3343c660286511afb1e3/staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go#L189-L192
Should we change them together in this PR or keep it as is to focus on bugfix only?

Reply:
another commit for v2 would be great.